### PR TITLE
Update release notes draft to version v1.30.0-alpha.3

### PR DESCRIPTION
**Pull request by npolshakova**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.30.0-alpha.3

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

**Review by puerco — APPROVED** [review]
LGTM, feel free to merge when ready!
/hold

**Comment by fykaa** [comment]
It appears that this pull request lacks maps &/or sessions. Ref: https://github.com/kubernetes/sig-release/pull/2278

**Comment by npolshakova** [comment]
Closing to rerun per: https://kubernetes.slack.com/archives/CN1KH4K9A/p1709658172605459?thread_ts=1709327978.866689&cid=CN1KH4K9A

**Pull request closed without merging**
